hack/testing/init-log-stack

```diff
@@ -33,6 +33,7 @@ openshift_logging_image_prefix=$imageprefix
 openshift_logging_use_ops=${ENABLE_OPS_CLUSTER:-False}
 openshift_logging_fluentd_use_journal=${USE_JOURNAL:-}
 openshift_logging_fluentd_journal_read_from_head=${JOURNAL_READ_FROM_HEAD:-False}
+openshift_logging_es_log_appenders=['file','console']
```

Comment:
do we want to have both file and console? can't we get away with just using 'console' for this PR?